.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "🚀 Feature request"
+about: Suggest an idea for this project
+labels: kind/feature
+---
+
+# Is your feature request related to a problem ?
+
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
I think E.g. is more commonly applicable than Ex.

Reply:
Fixed too